opl_storybook.module

```diff
@@ -166,28 +166,33 @@ function opl_storybook_block_view($delta = '') {
       }
 
       $chapterparentname = $treeterms[$chapterparentindex]->name;
-      
-      $prevchapterid = $treeterms[$chapterindex -1]->tid;
-      $prevchapterparent = $treeterms[$chapterindex -1]->parents[0];
-      $prevchaptername = $treeterms[$chapterindex -1]->name;
-      
-      $nextchapterid = $treeterms[$chapterindex +1]->tid;
-      $nextchapterparent = $treeterms[$chapterindex +1]->parents[0];
-      $nextchaptername = $treeterms[$chapterindex +1]->name;
 
-      $prevlink = l(t($prevchaptername), 'storybook/' . $chapterparentname . '/' . $prevchaptername);
-      $nextlink = l(t($nextchaptername), 'storybook/' . $chapterparentname . '/' . $nextchaptername);
+      $view_prev_next_content = '';
 
+      if( isset($treeterms[$chapterindex -1]) ){
+        $prevchapterid = $treeterms[$chapterindex -1]->tid;
+        $prevchapterparent = $treeterms[$chapterindex -1]->parents[0];
```

Comment:
Check to make sure parent[0] exist before fetching it, or else this could cause a warning.